gocourse08/collartask/animal/animal.go

```diff
@@ -0,0 +1,49 @@
+package animal
+
+type AnimalType string
+
+const (
+	Bear  AnimalType = "bear"
+	Tiger AnimalType = "tiger"
+	Lion  AnimalType = "lion"
+)
+
+//type Animal interface {
+//	Typify() string
+//	RecordBreathing(breathingData float64)
+//	RecordSound(soundData float64)
+//}
+
+type DataAnimal[T float64] struct {
+	AnimalType  string
+	Pulse       int
+	Temperature float64
+	Breaths     []T
+	Sounds      []T
+}
+
+func New(animalPulse int, animalTemperature float64, animalType AnimalType) DataAnimal[float64] {
```

Comment:
Простіше:
```suggestion
func New(pulse int, temperature float64, animalType AnimalType) DataAnimal[float64] {
```

З `animalType` не вийде змінити на `type`, тому хай буде так.